Client/mods/deathmatch/logic/luadefs/CLuaVehicleDefs.h

```diff
@@ -59,6 +59,8 @@ class CLuaVehicleDefs : public CLuaDefs
     LUA_DECLARE(GetVehicleNameFromModel);
     LUA_DECLARE(GetVehicleAdjustableProperty);
     LUA_DECLARE(GetHelicopterRotorSpeed);
+    //LUA_DECLARE(GetVehicleRotorSpeed); use new parser instead
+    static std::variant<bool,float> GetVehicleRotorSpeed(CClientVehicle* pVehicle);
```

Comment:
New functions do not need to point out that they use the new argument parser in a comment; they just can use it directly.

Also, I think you might forgot to run the clang formatter, as it'd format some return types with an additional space.

```suggestion
    static std::variant<bool, float> GetVehicleRotorSpeed(CClientVehicle* pVehicle);
```